flexmeasures/data/services/scheduling.py

```diff
@@ -245,6 +267,144 @@ def create_scheduling_job(
     return job
 
 
+def cb_done_sequential_scheduling_job(jobs_ids: list[str]):
+    """
+    TODO: add logic
+    """
+
+    # jobs = [Job.fetch(job_id) for job_id in jobs_ids]
+    pass
+
+
+def create_sequential_scheduling_job(
+    asset: Asset,
+    job_id: str | None = None,
+    enqueue: bool = True,
+    requeue: bool = False,
+    force_new_job_creation: bool = False,
+    scheduler_specs: dict | None = None,
+    depends_on: list[Job] | None = None,
+    **scheduler_kwargs,
```

Comment:
`scheduler_kwargs` is in desperate need of a docstring example, especially for its `flex_model`, which is partially deserialized after passing through `SequentialFlexModelSchema`.